Classes/Hooks/WizardItemsHookSubscriber.php

```diff
@@ -146,7 +146,7 @@ protected function getWhiteAndBlackListsFromPageAndContentColumn($pageUid, $colu
 		// Grid returned by the Provider that applies to the parent element's type and configuration
 		// (admitted, that's quite a mouthful - but it's not that different from reading the values from
 		// a page template like above; it's the same principle).
-		if (0 < $parentRecordUid && FALSE === empty($fluxAreaName)) {
+		if (isset($parentRecordUid) && 0 < $parentRecordUid && FALSE === empty($fluxAreaName)) {
```

Comment:
I would initialise `$parentRecordUid = 0` up where `defaultValues` is set, then leave this condition as-is. Should be safe that way and we avoid a potentially undeclared variables.